### PR TITLE
#149 Update Constitution links and pdf

### LINKED ISSUE #149: Outdated consitution
The constitution accessible via cssbham.com/constitution.pdf is outdated and the Google Doc link in /about is also outdated and, moreover, there is no reason to include it on the website.
I don't think we actually have the up-to-date constitution stored anywhere at the moment, just a Google Doc that has the changes highlighted.

- [ ] Create clean constitution document
- [ ] Upload pdf to website as `constitution.pdf`
- [ ] Add a redirect from cssbham.com/constitution
- [ ] Remove the Google Doc link in /about

### PR DESCRIPTION
**Pull request by sapphyree**
Update CSS Constitution file and clean up links on about as well as adding a /constitution redirect so it can be directly used instead of using /constitution.pdf. Closes #149.

### CODE REVIEW

**Review by AlexJBanks — CHANGES_REQUESTED** [review]
Given that there's only one link to our Constitution now, can the Constitution section of `/about` section be reworded to 
> That means we've got a constitution that you can **_read here_**. We recommend that all our members read it at least once.

**Comment by sapphyree** [comment]
> Given that there's only one link to our Constitution now, can the Constitution section of `/about` section be reworded to 
> > That means we've got a constitution that you can **_read here_**. We recommend that all our members read it at least once. 

I'm not entirely sure I agree with this. Part of me prefers the fact that it's sectioned off and very clearly visible. It being a hyperlink in a sentence feels to me like it'll be ignored by those reading over the about section.

I can do this if you're absolutely sure, though. Perhaps there's another solution as well?

**Comment by LikkanChung** [comment]
> > Given that there's only one link to our Constitution now, can the Constitution section of `/about` section be reworded to 
> > > That means we've got a constitution that you can **_read here_**. We recommend that all our members read it at least once. 
> 
> I'm not entirely sure I agree with this. Part of me prefers the fact that it's sectioned off and very clearly visible. It being a hyperlink in a sentence feels to me like it'll be ignored by those reading over the about section.
> 
> I can do this if you're absolutely sure, though. Perhaps there's another solution as well? 

I don't think it matters either way

**Review by LikkanChung — APPROVED** [review]

**Comment by Danyc0** [comment]
i agree with @sapphyree, it's nice that it's made obvious and clearly visible by being in its own section

**Comment by AlexJBanks** [comment]
Can we find a suitable compromise then, at the moment I'm not that happy with it just being a link on a single bullet point doing looking a bit sad on its own.



Could we turn the Constitution and Agendas links into buttons?

**Comment by Jack-Development** [comment]
We coud potenitally change the text to read like "Here is our constitution and agendas for you to read" and reuse the styles of the buttons higher up?

File: about.html [hunk 1]
```diff
@@ -101,17 +101,23 @@ <h4><i class="fab fa-github"></i> GitHub</h4>
...
-    <p>That means we've got a constitution that you can find below 👇</p>
-    <ul>
-        <li><a href="https://docs.google.com/document/d/1J3_EWs3dd2gq5T_xYylcP5pDzm0HHAYCTgiqKUYwkls">Constitution —
-            Google Doc</a></li>
-        <li><a href=/constitution.pdf>Constitution — PDF</a></li>
-    </ul>
-    <p>We recommend that all our members read it at least once. </p>
-    <p> You can also find our agendas <a href=/agendas">here</a>. </p>
+    <div id="constitution">
+        <div class="flex-container flex-v-center">
```
Comment (AlexJBanks) [inline about.html:105]: these div are probably redundant but can be tidyied up in #153

**Review by AlexJBanks — APPROVED** [review]

**Pull request merged**